packages/webapp/src/components/Task/PureIrrigationTask/index.jsx

```diff
@@ -197,7 +197,7 @@ export default function PureIrrigationTask({
   const waterCalculatorDisabled =
     !showWaterUseCalculatorModal ||
     !measurement_type ||
-    (measurement_type === 'DEPTH' && !location);
+    (measurement_type === 'DEPTH' && !(location || getValues().show_wild_crop));
```

Comment:
Using this form value works on task creation, but if the user wants to open the calculator when they complete the task, `show_wild_crop` won't be available -- I think it's unique to creation and maybe something else like `pinCoordinates` would have to control the logic for task completion?